<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FCWindow</class>
 <widget class="QDialog" name="FCWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>160</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>401</width>
    <height>160</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>401</width>
    <height>160</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>401</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FCWindow</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>153</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz katalog z plikami</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_subdirs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>przeszukaj podkatalogi</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_resize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>147</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>zmniejsz rozmiar do:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_size">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2JPG">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>129</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>konwertuj do JPG</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>px</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_wybierz">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>30</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Wybierz</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_greyscale">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>do greyscale</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dir">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>301</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="znaleziono">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Znaleziono</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_resize</sender>
   <signal>stateChanged(int)</signal>
   <receiver>lineEdit_size</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
